src/integration-tests/src/sql.rs

```diff
@@ -0,0 +1,136 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::{Error, RandomChars, Result};
+use rand::Rng;
+use sql::model;
+
+pub async fn run_sql(builder: sql::builder::sql_instances_service::ClientBuilder) -> Result<()> {
+    // Enable a basic subscriber. Useful to troubleshoot problems and visually
+    // verify tracing is doing something.
+    #[cfg(feature = "log-integration-tests")]
+    let _guard = {
+        use tracing_subscriber::fmt::format::FmtSpan;
+        let subscriber = tracing_subscriber::fmt()
+            .with_level(true)
+            .with_thread_ids(true)
+            .with_span_events(FmtSpan::NEW | FmtSpan::CLOSE)
+            .finish();
+
+        tracing::subscriber::set_default(subscriber)
+    };
+
+    let project_id = crate::project_id()?;
+    let name = random_sql_instance_name(&project_id);
+    let client: sql::client::SqlInstancesService = builder.build().await?;
+
+    cleanup_stale_sql_instances(&client, &project_id).await?;
+
+    println!("\nTesting insert sql instance");
+    let insert = client
```

Comment:
It would be nice if the tests included any attribute that uses the `json_name` attribute. For example:

https://github.com/googleapis/googleapis/blob/351a2dc65443e3fd50280a372830ba963699cfa6/google/cloud/sql/v1/cloud_sql_tiers.proto#L68

Maybe in the `get` request you can check the value of `RAM`?